diff -up kwindowsystem-5.99.0/CMakeLists.txt.1~ kwindowsystem-5.99.0/CMakeLists.txt
--- kwindowsystem-5.99.0/CMakeLists.txt.1~	2022-10-10 19:35:00.139959435 +0200
+++ kwindowsystem-5.99.0/CMakeLists.txt	2022-10-10 19:35:46.746221280 +0200
@@ -58,10 +58,6 @@ if (APPLE)
     cmake_find_frameworks(CoreFoundation)
 endif()
 
-if(WIN32 AND QT_MAJOR_VERSION EQUAL "5")
-   find_package(Qt5WinExtras REQUIRED)
-endif()
-
 if (NOT APPLE)
     find_package(X11)
 endif()
diff -up kwindowsystem-5.99.0/KF5WindowSystemConfig.cmake.in.1~ kwindowsystem-5.99.0/KF5WindowSystemConfig.cmake.in
--- kwindowsystem-5.99.0/KF5WindowSystemConfig.cmake.in.1~	2022-10-01 16:26:14.000000000 +0200
+++ kwindowsystem-5.99.0/KF5WindowSystemConfig.cmake.in	2022-10-10 19:35:00.139959435 +0200
@@ -12,10 +12,6 @@ if(@X11_FOUND@)
 endif()
 
 if(NOT @BUILD_SHARED_LIBS@)
-    if(@WIN32@)
-        find_dependency(Qt5WinExtras @REQUIRED_QT_VERSION@)
-    endif()
-
     if(@X11_FOUND@)
         if (NOT TARGET Qt6::Gui)
             find_dependency(Qt5X11Extras @REQUIRED_QT_VERSION@)
