diff -up kwindowsystem-5.88.0/CMakeLists.txt.1~ kwindowsystem-5.88.0/CMakeLists.txt
--- kwindowsystem-5.88.0/CMakeLists.txt.1~	2021-11-06 14:26:26.000000000 +0100
+++ kwindowsystem-5.88.0/CMakeLists.txt	2021-11-20 12:05:19.420628281 +0100
@@ -58,10 +58,6 @@ if (APPLE)
     cmake_find_frameworks(CoreFoundation)
 endif()
 
-if(WIN32)
-   find_package(Qt5WinExtras REQUIRED)
-endif()
-
 if (NOT APPLE)
     find_package(X11)
 endif()
diff -up kwindowsystem-5.88.0/KF5WindowSystemConfig.cmake.in.1~ kwindowsystem-5.88.0/KF5WindowSystemConfig.cmake.in
--- kwindowsystem-5.88.0/KF5WindowSystemConfig.cmake.in.1~	2021-11-20 12:05:19.420628281 +0100
+++ kwindowsystem-5.88.0/KF5WindowSystemConfig.cmake.in	2021-11-20 12:06:48.784851572 +0100
@@ -12,10 +12,6 @@ if(@X11_FOUND@)
 endif()
 
 if(NOT @BUILD_SHARED_LIBS@)
-    if(@WIN32@)
-        find_dependency(Qt5WinExtras @REQUIRED_QT_VERSION@)
-    endif()
-
     if(@X11_FOUND@)
         find_dependency(Qt5X11Extras @REQUIRED_QT_VERSION@)
         find_dependency(XCB)
