diff -up kwindowsystem-5.90.0/CMakeLists.txt.1~ kwindowsystem-5.90.0/CMakeLists.txt
--- kwindowsystem-5.90.0/CMakeLists.txt.1~	2022-01-01 13:37:48.000000000 +0100
+++ kwindowsystem-5.90.0/CMakeLists.txt	2022-01-07 01:31:21.434019037 +0100
@@ -58,10 +58,6 @@ if (APPLE)
     cmake_find_frameworks(CoreFoundation)
 endif()
 
-if(WIN32)
-   find_package(Qt5WinExtras REQUIRED)
-endif()
-
 if (NOT APPLE)
     find_package(X11)
 endif()
diff -up kwindowsystem-5.90.0/KF5WindowSystemConfig.cmake.in.1~ kwindowsystem-5.90.0/KF5WindowSystemConfig.cmake.in
--- kwindowsystem-5.90.0/KF5WindowSystemConfig.cmake.in.1~	2022-01-07 01:31:21.434019037 +0100
+++ kwindowsystem-5.90.0/KF5WindowSystemConfig.cmake.in	2022-01-07 01:59:57.791809398 +0100
@@ -12,10 +12,6 @@ if(@X11_FOUND@)
 endif()
 
 if(NOT @BUILD_SHARED_LIBS@)
-    if(@WIN32@)
-        find_dependency(Qt5WinExtras @REQUIRED_QT_VERSION@)
-    endif()
-
     if(@X11_FOUND@)
         if (NOT TARGET Qt6::Gui)
             find_dependency(Qt5X11Extras @REQUIRED_QT_VERSION@)
